client/config.go

```diff
@@ -37,6 +38,7 @@ func newDefaultConfig() *config {
 
 	return &config{
 		url:          "http://localhost:7999/api/1.0",
+		eventsURL:    "http://localhost:7998/api/1.0",
```

Comment:
I think we need to set this to the UAT URLs (though we can hold off as it will need set to the production URL post GA).